gazebo:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 500

  # Effort Controllers ---------------------------------------
  preshape_j1_position_controller:
    type: effort_controllers/JointPositionController
    joint: preshape_1
    pid: {p: 1.0, i: 0.0, d: 0.0}
  preshape_j2_position_controller:
    type: effort_controllers/JointPositionController
    joint: preshape_2
    pid: {p: 1.0, i: 0.0, d: 0.0}
  proximal_j1_position_controller:
    type: effort_controllers/JointPositionController
    joint: proximal_joint_1
    pid: {p: 1.0, i: 0.0, d: 0.0}
  proximal_j2_position_controller:
    type: effort_controllers/JointPositionController
    joint: proximal_joint_2
    pid: {p: 1.0, i: 0.0, d: 0.0}
  proximal_j3_position_controller:
    type: effort_controllers/JointPositionController
    joint: proximal_joint_3
    pid: {p: 1.0, i: 0.0, d: 0.0}
